code/modules/mob/mob.dm

```diff
@@ -533,7 +533,12 @@
 	else
 		result = examinify.examine(src) // if a tree is examined but no client is there to see it, did the tree ever really exist?
 
-	to_chat(src, "<span class='infoplain'>[result.Join("\n")]</span>")
+	if(result.len)
+		for(var/i = 1, i <= result.len, i++)
+			if(!findtext(result[i], "<hr>"))
+				result[i] += "\n"
+
+	to_chat(src, "<div class='examine_block'><span class='infoplain'>[result.Join()]</span></div>")
```

Comment:
```suggestion
	for(var/i in 1 to length(result))
		if(!findtext(result[i], "<hr>"))
			result[i] += "\n"
	to_chat(src, "<div class='examine_block'><span class='infoplain'>[result.Join()]</span></div>")
```
for(var/i in x to y) is faster, nicer, and functionally equivalent if the list's size doesn't change
to remove the trailing new line, do length(result) - 1